### PR TITLE
STORM-2190: reduce contention between submission and scheduling

### PR DESCRIPTION
**Pull request by revans2**

### CODE REVIEW

File: examples/storm-starter/src/jvm/org/apache/storm/starter/WordCountTopology.java [hunk 1]
```diff
@@ -90,7 +90,9 @@ public static void main(String[] args) throws Exception {
     if (args != null && args.length > 0) {
       conf.setNumWorkers(3);
 
-      StormSubmitter.submitTopologyWithProgressBar(args[0], conf, builder.createTopology());
+      for (String name: args) {
```
Comment (ptgoetz) [inline examples/storm-starter/src/jvm/org/apache/storm/starter/WordCountTopology.java]: I'm not sure I understand the for loop here. Isn't this change just submitting the same topology multiple times with different names? Is this an artifact from manual testing?

**Comment by ptgoetz** [comment]
One question about the for loop in WordCountTopology. I'm +1 once that's answered/addressed.

File: examples/storm-starter/src/jvm/org/apache/storm/starter/WordCountTopology.java (hunk 1 above)
Comment (revans2) [inline examples/storm-starter/src/jvm/org/apache/storm/starter/WordCountTopology.java]: This was code that I put in to be able to reproduce the problem consistently.  If you want me to remove it I am happy to.  It does not really provide any real value beyond that.

**Comment by ptgoetz** [comment]
+1

**Pull request merged**